common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java

```diff
@@ -1388,46 +1451,56 @@
          */
         public static final String AUTHORITY = "microsoft.identity.broker";
 
+        private static final String VERSION_1 = ONE_POINT_ZERO;
+        private static final String VERSION_5 = FIVE_POINT_ZERO;
+        private static final String VERSION_6 = "6.0";
+        private static final String VERSION_7 = SEVEN_POINT_ZERO;
+        private static final String BROKER_VERSION_1 = ONE_POINT_ZERO;
+        private static final String BROKER_VERSION_2 = TWO_POINT_ZERO;
+
         /**
          * Tie the API paths and codes into a single object structure to stop us from having to keep
-         * them in sync.
-         */
+         * them in sync.  This is designed to pull all the parts of the API definition into a single
+         * place, so that we only need to make updates in one location, and it's clearer what we need
+         * to do when adding new APIs.
+         *
+         *
+         */
+        @Getter
+        @Accessors(prefix = "m")
+        @AllArgsConstructor
         public enum API {
-            MSAL_HELLO(MSAL_HELLO_PATH, MSAL_HELLO_URI_CODE),
-            ACQUIRE_TOKEN_INTERACTIVE(MSAL_ACQUIRE_TOKEN_INTERACTIVE_PATH, MSAL_ACQUIRE_TOKEN_INTERACTIVE_CODE),
-            ACQUIRE_TOKEN_SILENT(MSAL_ACQUIRE_TOKEN_SILENT_PATH, MSAL_ACQUIRE_TOKEN_SILENT_CODE),
-            GET_ACCOUNTS(MSAL_GET_ACCOUNTS_PATH, MSAL_GET_ACCOUNTS_CODE),
-            REMOVE_ACCOUNTS(MSAL_REMOVE_ACCOUNTS_PATH, MSAL_REMOVE_ACCOUNTS_CODE),
-            GET_CURRENT_ACCOUNT_SHARED_DEVICE(MSAL_GET_CURRENT_ACCOUNT_SHARED_DEVICE_PATH, MSAL_GET_CURRENT_ACCOUNT_SHARED_DEVICE_CODE),
-            GET_DEVICE_MODE(MSAL_GET_DEVICE_MODE_PATH, MSAL_GET_DEVICE_MODE_CODE),
-            SIGN_OUT_FROM_SHARED_DEVICE(MSAL_SIGN_OUT_FROM_SHARED_DEVICE_PATH, MSAL_SIGN_OUT_FROM_SHARED_DEVICE_CODE),
-            GENERATE_SHR(GENERATE_SHR_PATH, MSAL_GENERATE_SHR_CODE),
-            BROKER_HELLO(BROKER_API_HELLO_PATH, BROKER_API_HELLO_URI_CODE),
-            BROKER_GET_ACCOUNTS(BROKER_API_GET_BROKER_ACCOUNTS_PATH, BROKER_API_GET_BROKER_ACCOUNTS_CODE),
-            BROKER_REMOVE_ACCOUNT(BROKER_API_REMOVE_BROKER_ACCOUNT_PATH, BROKER_API_REMOVE_BROKER_ACCOUNT_CODE),
-            BROKER_UPDATE_BRT(BROKER_API_UPDATE_BRT_PATH, BROKER_API_UPDATE_BRT_CODE),
-            BROKER_ADD_FLIGHTS(BROKER_API_ADD_FLIGHTS_PATH),
-            BROKER_SET_FLIGHTS(BROKER_API_SET_FLIGHTS_PATH),
-            BROKER_GET_FLIGHTS(BROKER_API_GET_FLIGHTS_PATH),
-            GET_SSO_TOKEN(GET_SSO_TOKEN_PATH),
-            UNKNOWN(null)
+            MSAL_HELLO(MSAL_HELLO_PATH, null, VERSION_5),
```

Comment:
Should these be 3?

ref: https://identitydivision.visualstudio.com/DevEx/_git/AuthLibrariesApiReview?path=%2F%5BAndroid%5D%20Broker%20API%2Fbroker_protocol_versions.md

Reply:
fixed.  I plan to copy or this into the ad-accounts package at some point.